### PR TITLE
Add forward compatibility for PatientStatus

### PR DESCRIPTION
**Pull request by sanchita**
https://www.pivotaltracker.com/story/show/167163204

### CODE REVIEW

File: app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt [hunk 1]
```diff
@@ -2357,6 +2357,73 @@ class DatabaseMigrationAndroidTest {
     verifyNewAppointmentStatus(uuid = uuid2, newStatus = "cancelled")
     verifyNewAppointmentStatus(uuid = uuid3, newStatus = "visited")
   }
+
+  @Test
+  fun migrate_patient_status_from_38_to_39() {
+    val db_38 = helper.createDatabase(version = 38)
+
+
```
Comment (saket) [inline app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt]: 😜

File: app/src/main/java/org/simple/clinic/storage/Migration_38_39.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+@Suppress("ClassName")
+class Migration_38_39 : Migration(38, 39) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.execSQL("""
+      UPDATE "Patient"
+      SET "status" = (
+        CASE
```
Comment (saket) [inline app/src/main/java/org/simple/clinic/storage/Migration_38_39.kt]: Nice!

File: app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt (hunk 1 above)
Comment (sanchita) [reply app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt]: Stop with this nit-pick :P

Comment (saket) [reply app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt]: Sorry, I was just pulling your leg :p

**Pull request merged**